libraries/Microsoft.Bot.Builder/BotState.cs

```diff
@@ -302,16 +293,22 @@ public Task DeleteAsync(ITurnContext turnContext)
             /// </summary>
             /// <param name="turnContext">turn context</param>
```

Comment:
><param name="turnContext">turn context</param> [](start = 16, length = 46)

add param description of defaultValueFactory

Reply:
Nice catch.